interfaces/connection.go

```diff
@@ -49,29 +50,49 @@ type ConnectedSlot struct {
 // Attrer is an interface with Attr getter method common
 // to ConnectedSlot, ConnectedPlug, PlugInfo and SlotInfo types.
 type Attrer interface {
-	Attr(key string, val interface{}) error
```

Comment:
general doc needs updating and probably each interface method needs its own bit of doc

Reply:
Done.